AmplifyTestCommon/Models/TransformerV2/4aV2/Project4aV2+Schema.swift

```diff
@@ -31,7 +31,7 @@ extension Project4aV2 {
     model.fields(
       .id(),
       .field(project4aV2.name, is: .optional, ofType: .string),
-      .hasOne(project4aV2.team, is: .optional, ofType: Team4aV2.self, associatedWith: Team4aV2.keys.project, targetName: "project4aV2TeamId"),
+      .hasOne(project4aV2.team, is: .optional, ofType: Team4aV2.self, associatedWith: Team4aV2.keys.id, targetName: "project4aV2TeamId"),
```

Comment:
AmplifyTestCommon files are used by unit tests. is this change expected? If so, would prefer to make this change outside this PR with the details on the issue